miniconda/debian/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+. /opt/conda/etc/profile.d/conda.sh && conda activate base
```

Comment:
I'm a little rusty on Docker, but this should get run once at the beginning of the sort-of docker session, right?

I was a little worried it would force the userinto the `base` conda env on each line's execution, thus preventing the creation of new environments. But that's probably not a danger here, right?

Reply:
> I was a little worried it would force the userinto the base conda env on each line's execution, thus preventing the creation of new environments

Not sure what you mean by this? Auto activating the base env in no way impacts the ability create new envs, in fact it is a requirement that the base env is activated before creating new environments ...

Reply:
.. adding something to PATH before appeared to suffice, but if this had been Windows then that strategy would have been very messy (requiring 6 entires to PATH in the right order with the correct capitalization).